### ISSUE TITLE
Dashboards: InfluxDB Flux query dashboard variables fail since v11.1

### ISSUE THREAD

**Issue by tvdhout** [grafana/grafana#88983]
### What happened?

Since Grafana v11.1, my Flux query dashboard variable fails when retrieving its values upon loading the dashboard.

My variable query:
```
import "influxdata/influxdb/schema"

schema.tagValues(bucket: "my_bucket", tag: "my_tag", start: v.timeRangeStart, stop: v.timeRangeStop)
```
This query should retrieve all unique tag values in the specified time range.

The v.timeRangeStart/Stop values are replaced with 1970-01-01T00.00.00Z (timestamp 0), regardless of the dashboard's selected time range. In queries sent from dashboard visualizations, additional values for "from" and "to" are added to the HTTP request that queries the database. The "from" and "to" keys are missing from requests sent for the variable. This results in a status 400 HTTP response with the error: This results in a status 400 response with the message `“error”: “invalid: error in building plan while starting program: cannot query an empty range”`. 

The same query I use to retrieve the variable values _does_ work in the data explorer, so it's specific to flux query variables.

Others also have this issue, see my [Grafana Community Forum post](https://community.grafana.com/t/time-range-values-not-passed-to-variable-queries/123269)

It may be related to https://github.com/grafana/grafana/pull/88642

### What did you expect to happen?

Load the unique tag values from the provided tag in the provided time range.

### Did this work before?

Yes, it fails since June 4 or 5. It worked fine before on this same dashboard. Nothing changed in the datasource or dashboard.

### How do we reproduce it?

1. Create an InfluxDB datasource with some measurements with a tag with varying values
2. Create a dashboard variable with my above query (alter the bucket and tag names)
3. Try to use variable in a dashboard visualization.
4. Check the HTTP requests sent in your browser tools

### Environment (with versions)?

Grafana: Grafana v11.1.0-71516 (0ba54152dc)
OS: MacOS Sonoma 14.5 (23F79)
Browser: Safari (bug is not browser-specific)

### Grafana platform?

I use Grafana Cloud

### Datasource(s)?

InfluxDB v2.7.4

**Comment by rbagd** [comment]
We are experiencing the same issue on our Grafana Cloud instance. Flux queries which were working before are now returning HTTP 400 with empty time range.

**Comment by itsmylife** [comment]
Thanks for reporting the issue. I found the problem and I will it today.

**Comment by jtp10181** [comment]
I see this fix was backported to 11.1, how do get the current code via the apt repo?
